crates/gpui/src/platform/common/text_system.rs

```diff
@@ -228,8 +228,15 @@ impl LinuxTextSystemState {
                 .get_font(font_id)
                 .ok_or_else(|| anyhow!("Could not load font"))?;
 
-            // HACK: to let the storybook run, we should actually do better font fallback
-            if font.as_swash().charmap().map('m') == 0 || postscript_name == "Segoe Fluent Icons" {
+            // HACK: To let the storybook run and render Windows caption icons. We should actually do better font fallback.
+            let allowed_bad_font_names = vec![
+                "SegoeFluentIcons", // NOTE: Segoe fluent icons postscript name is inconsistent
+                "Segoe Fluent Icons",
+            ];
```

Comment:
this is my finding while running on my system with the pre installed segoe fluent icons on Windows

Reply:
This seems to be the case with a few others as well -- at least Intel One Mono comes to mind as one that is known to have used both "IntelOneMono" and "Intel One Mono". (IIRC they originally used the former and then fixed it to the latter in an update)

Maybe the code could just silently retry with all whitespace removed from the name? And potentially log a note if it finds the font with a different name. Maybe even show it as a hint when editing the config file.

Reply:
> Maybe the code could just silently retry with all whitespace removed from the name? And potentially log a note if it finds the font with a different name. Maybe even show it as a hint when editing the config file.

I think explicitly enumerating the variants we want to try will be better, at least until we better understand the impact.